modules/Forum/module.php

```diff
@@ -244,9 +249,17 @@ public function onPageLoad($user, $pages, $cache, $smarty, $navs, $widgets, $tem
         }
 
         // Widgets
-        if (defined('FRONT_END') || (defined('PANEL_PAGE') && str_contains(PANEL_PAGE, 'widget'))) {
+        if ($pages->getActivePage()['widgets'] || (defined('PANEL_PAGE') && str_contains(PANEL_PAGE, 'widget'))) {
```

Comment:
This change means we don't do `n` queries on every frontend page if they didn't support widgets.
For example, we loaded all widgets on the status page, even though it doesn't support widgets.